dali/pipeline/data/tensor_list.h

```diff
@@ -548,7 +518,7 @@ class DLL_PUBLIC TensorList : private Buffer<Backend> {
     // need to create a new view
     DALI_ENFORCE(num_samples() > 0,
                  "To create a view Tensor, the Tensor List must have at least 1 element.");
-    DALI_ENFORCE(IsValidType(type()),
+    DALI_ENFORCE(IsValidType(type_info()),
```

Comment:
```suggestion
    DALI_ENFORCE(IsValidType(type()),
```
Don't use type_info for such a simple check.

Reply:
Done